app/views/items/_item_grid.html.erb

```diff
@@ -0,0 +1,20 @@
+<div id="items">
+  <div class="container">
+      <div class="row ">
+        <% @items.each do |item| %>
+          <div class="col-sm-3 mb-4">
+          <div class="card">
+            <img class="card-img-top" src="https://via.placeholder.com/400x300" alt="Card image cap">
+            <div class="card-body text-center">
+              <h5 class="card-title text-truncate"><%= item.name %></h5>
+              <p class="card-text text-truncate">von: Alan Must</p>
```

Comment:
- question: what is the purpose/expected life time of these "example data"? If there are no data yet, there should be no data displayed.